app.js

```diff
@@ -26,3 +29,9 @@ const productsCenterDOM = document.querySelector(".products-center");
   }
   productsCenterDOM.innerHTML = results;
 })();
+
+cartDOM.addEventListener("click", showCart);
+function showCart() {
```

Comment:
Since, you don't call this method, please add suffix of "Handler" to make it more understandable.

Reply:
Thanks for your kind concern , i made the changes you suggested and the output was fine.